spec/models/amusement_park_spec.rb

```diff
@@ -1,7 +1,24 @@
 require "rails_helper"
 
 RSpec.describe AmusementPark, type: :model do
+  before(:each) do
+    @mechanic = Mechanic.create(name: "Kara Smith", years_experience: 11)
+    @mechanic2 = Mechanic.create(name: "Cara Smith", years_experience: 12)
+    @amusement_park = AmusementPark.create(name: 'Six Flags', admission_cost: 75)
+    @ride = @amusement_park.rides.create(name: "The Hurler", thrill_rating: 7, open: false)
+    @ride2 = @amusement_park.rides.create(name: "Swift", thrill_rating: 10, open: true)
+    @ride3 = @amusement_park.rides.create(name: "Boom", thrill_rating: 9, open: true)
+    MechanicRide.create(ride_id: @ride.id, mechanic_id: @mechanic.id)
+    MechanicRide.create(ride_id: @ride3.id, mechanic_id: @mechanic2.id)
+  end
+
   describe "relationships" do
     it { should have_many(:rides) }
```

Comment:
Once you add that additional association to mechanics, you can add 
`it { should have_many(:mechanics).through(:rides) }`